src/commonMain/kotlin/exchange.dydx.abacus/state/model/TradingStateMachine+Wallet.kt

```diff
@@ -162,9 +171,14 @@ internal fun TradingStateMachine.receivedFills(
     payload: Map<String, Any>,
     subaccountNumber: Int,
 ): StateChanges {
-    val size = parser.asList(payload["fills"])?.size ?: 0
+    val fills = parser.asList(payload["fills"])
+    val size = fills?.size ?: 0
     return if (size > 0) {
-        wallet = walletProcessor.receivedFills(wallet, payload, subaccountNumber)
+        wallet = walletProcessor.receivedFillsDeprecated(wallet, payload, subaccountNumber)
+        if (staticTyping) {
+            val payload = Json.decodeFromString<IndexerFillResponse?>(payload.toJson())
```

Comment:
try catch

Reply:
Jeremy had a good suggestion before that it might be useful to create a helper function around `Json.decodeFromString` with a try-catch. Then we can forbid usages of it directly via Detekt.